metricbeat/module/linux/linux.go

```diff
@@ -33,7 +33,7 @@ func init() {
 // Module defines the base module config used in `linux`
 type Module struct {
 	mb.BaseModule
-	HostFS string `config:"hostfs"`
+	HostFS string `config:"hostFS"`
```

Comment:
Hmmm, I did a quick check across the Metricbeat codebase and I don't see any config setting named with uppercase characters in it. So I think we might want to leave the name of the config setting as `hostfs`. My earlier comment was referring to the identifiers in the code (struct fields, variables, etc.)